easybuild/easyconfigs/c/CellRank/CellRank-2.0.2-foss-2022a.eb

```diff
@@ -0,0 +1,67 @@
+easyblock = 'PythonBundle'
+
+name = 'CellRank'
+version = '2.0.2'
+
+homepage = 'https://cellrank.readthedocs.io/en/stable/'
+description = """CellRank is a toolkit to uncover cellular dynamics based on
+ Markov state modeling of single-cell data.  It contains two main modules:
+kernels compute cell-cell transition probabilities and estimators generate
+hypothesis based on these. """
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('petsc4py', '3.17.4'),
+    ('slepc4py', '3.17.2'),
+    ('scikit-learn', '1.1.2'),
+    ('scVelo', '0.2.5'),
```

Comment:
I'll look into updating this to the latest version (0.3.1)

Reply:
I spent quite a bit of time on this, because `scVelo` 0.3.1 introduces a new requirement on `scvi-tools`, which is a bit of a nightmare for a variety of reasons (like scvi-tools 0.20.3 already requiring jax 0.4.4 or newer, see #17758), so I gave up on it.

For the `2022a` generation, sticking to `scVelo` 0.2.5 is reasonable though imho...